package.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
Let's add [engines](https://files.slack.com/files-pri/T02EKBW5B-F03S35LGMHU/image.png) to specify node and npm version.

Reply:
Added in [adc7753](https://github.com/rosle/react-nimble-survey/pull/9/commits/adc77533885dae70bb6ecf05fefbe95041fceff0) 👍 

For this, in which case do you see the benefits, Micky? As I read, it is to provide the compatibilities range, correct? different from our `.tool-versions` which only lock the version in dev. 🤔 Wonder if it still useful for non-npm-published package like this 💭

Reply:
As I know, the only benefit is to document the compatible node and npm version in case other reviewers are not familiar with `.tool-versions` so they can check the `package.json` instead.

I noticed that I provided the wrong link to the doc 🙈 I updated the engines link in the original comment.